TDP-2470: support more verbs on Transfer B leg (#435)
Bandwidth/bandwidth.github.io: voice/bxml/callbacks/transferAnswer.md

{% method %}
##  Transfer Answer Event – <Transfer> verb
When processing a [`<Transfer>`](../verbs/transfer.md) verb, this event is sent when a called party (B-leg) answers.  The event is sent to

>   the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered. Accepted verbs returned by this callback will be

Reviewer: I would remove the word "accepted". The whole BXML document either gets executed or rejected. As written, it sounds kind of like allowed verbs will be executed and bad ones will be filtered out

  executed for the called party only. Afterward, the called party will be bridged to the original
  call.

> | Verb            | Accepted |

Reviewer: I think this table needs a little sentence introducing it or a title

|-----------------|----------|
| [Conference](../verbs/conference.md)      | No       |

> | [Forward](../verbs/forward.md)         | No       |

Reviewer: It might look better if this table was below the expected response and properties? Not sure

Author: I think it's better up top since it's more event-specific than the properties and expected response, since they're on every callback page and have a lot of overlap.

Reviewer: Yeah good point

| [Gather](../verbs/gather.md)          | Yes      |
| [Hangup](../verbs/hangup.md)          | Yes      |
| [Pause](../verbs/pause.md)           | Yes      |
| [PauseRecording](../verbs/pauseRecording.md)  | Yes      |
| [PlayAudio](../verbs/playAudio.md)       | Yes      |
| [Record](../verbs/record.md)          | Yes      |
| [Redirect](../verbs/redirect.md)        | Yes      |
| [ResumeRecording](../verbs/resumeRecording.md) | Yes      |
| [SendDtmf](../verbs/sendDtmf.md)        | Yes      |
| [SpeakSentence](../verbs/speakSentence.md)   | Yes      |
| [StartRecording](../verbs/startRecording.md)  | Yes      |
| [StopRecording](../verbs/stopRecording.md)   | Yes      |

> | [Transfer](../verbs/transfer.md)        | No       |

Reviewer: Spacing is weird in this table. Recommend either removing all the whitespace or lining it up. As it is there's some padding but it doesn't line up.

It is important to note that no other BXML verbs may be specified after a Transfer Answer Event is called.

### Expected response
```http
HTTP/1.1 200
Content-Type: application/xml; charset=utf-8

<Response>

>     <!-- Accepted BXML verbs to play to the called party -->

Reviewer: We can do a better example than this. Can you fill it in with an actual SpeakSentence or something?

Reviewer: Never mind, I just realized that this isn't an example, my bad. Ignore

{% method %}
##  Transfer Answer Event – <Transfer> verb
When processing a [`<Transfer>`](../verbs/transfer.md) verb, this event is sent when a called party (B-leg) answers.  The event is sent to

>   the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered. Valid BXML returned by this callback will be executed for the called party only. Afterward, the called party will be bridged to the original

Reviewer: ```suggestion
  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered. BXML returned by this callback will be executed for the called party only. After all BXML has been executed, the called party will be bridged to the original
```

  call. 
  

>   Valid Bxml must be made up of executable verbs. The list of verbs that are executable when returned from this callback is as follows: 

Reviewer: ```suggestion
  Most BXML verbs are allowed in response to a `transferAnswer` event, but some are not allowed. A full list of verbs and whether or not they are allowed:
```

> | Verb                                              | Executable    |

Reviewer: ```suggestion
| Verb                                              | Allowed       |
```

|---------------------------------------------------|---------------|
| [Conference](../verbs/conference.md)              | No            |
| [Forward](../verbs/forward.md)                    | No            |
| [Gather](../verbs/gather.md)                      | Yes           |
| [Hangup](../verbs/hangup.md)                      | Yes           |
| [Pause](../verbs/pause.md)                        | Yes           |
| [PauseRecording](../verbs/pauseRecording.md)      | Yes           |
| [PlayAudio](../verbs/playAudio.md)                | Yes           |
| [Record](../verbs/record.md)                      | Yes           |
| [Redirect](../verbs/redirect.md)                  | Yes           |
| [ResumeRecording](../verbs/resumeRecording.md)    | Yes           |
| [SendDtmf](../verbs/sendDtmf.md)                  | Yes           |
| [SpeakSentence](../verbs/speakSentence.md)        | Yes           |
| [StartRecording](../verbs/startRecording.md)      | Yes           |
| [StopRecording](../verbs/stopRecording.md)        | Yes           |
| [Transfer](../verbs/transfer.md)                  | No            |

> It is important to note that no other BXML verbs may be specified after a Transfer Answer Event is called.

Reviewer 2: I know you didn't write this, but what does this mean? Does it still apply?

Author: I believe so? I think it's just warning you if someone answers the transfer, no BXML after the transfer is going to be executed since it goes to whatever endpoint is specified as the transferAnswerURL.

Reviewer: I would just remove it. That's what the table is saying

### Expected response
```http
HTTP/1.1 200
Content-Type: application/xml; charset=utf-8

<Response>

>     <!-- Accepted BXML verbs to play to the called party -->

Reviewer 2: ```suggestion
    <!-- Valid, executable BXML verbs to play to the called party -->
```

| to               | The phone number used in the `to` field of the original call, in E.164 format (e.g. +15555555555).                |
| direction        | The direction of the call. Always `outbound` for this event.                                                      |
| callId           | The call id of the newly-created B leg.                                                                           |

> | parentCallId     | The call id of the original call leg that initiated the `<Transfer>` tag.                                         |

Reviewer: executed?
